### ISSUE TITLE
Update a landmark and add a new one

### ISSUE THREAD

**Issue by jvine** [ProjectMirador/mirador#2292]
The sidebar tabs area that was made a `<nav>` in #2236 is missing a label. Please add:
`aria-label="Sidebar panels navigation"` to the `<nav>` element.

The window top bar is missing a `<nav>` container. Please add:
`<nav aria-label="Window navigation">`
within the `<header>` element and surrounding the buttons.
